### DISCUSSION TITLE
Add a "used by"/"showcase" section?

### DISCUSSION THREAD

**Discussion by jasikpark** [PaloAltoNetworks/docusaurus-openapi-docs#464] [Ideas]
Would be cool to see where this project is being used, to see *how* it's being used :)

I'm also proud of the work @IanVS has done on our docs && want to link it here! <https://docs.defined.net/api>

This also might just be a small enough project where that's not needed but 🤷

**Comment by sserrata** [comment]
I agree, a "showcase" page or something similar would be nice - even better if it had a self-service way to add your site. 

I really like what @homotechsual did with https://docusaurus.community.

**Comment by IanVS** [comment]
A pinned issue where people can drop links is also a low-effort way to accomplish this.
